docker-compose.yaml

```diff
@@ -44,7 +44,7 @@ services:
         splinterd \
             -c ./configs/splinterd-node-0-docker.toml \
             --registry file://./node_registries/nodes.yaml \
-            --insecure \
+            --tls-insecure \
```

Comment:
Typos in commit message: s/not tls_ca_file/now tls_ca_file/ and s/ca_file -> tls_ca_file/ca_cert -> tls_ca_file/